src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -316,17 +316,11 @@ private async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var triggersArray = new JArray(triggers);
             int count = triggersArray.Count;
 
+            // add host configuration
             JObject hostConfig = null;
-            if (_environment.IsFlexConsumptionSku())
+            if (Utility.TryGetHostService<IHostOptionsProvider>(_scriptHostManager, out IHostOptionsProvider hostOptionsProvider))
             {
-                // TODO: Only adding host configuration for Flex. Once SyncTriggers is marked RequiresRunningHost,
-                // we'll enable across the board.
-                // https://github.com/Azure/azure-functions-host/issues/9904
-                // If an active host is running, add host configuration.
-                if (Utility.TryGetHostService<IHostOptionsProvider>(_scriptHostManager, out IHostOptionsProvider hostOptionsProvider))
-                {
-                    hostConfig = hostOptionsProvider.GetOptions();
-                }
+                hostConfig = hostOptionsProvider.GetOptions();
```

Comment:
Just clarifying -- we can expect a hostConfig attribute 80% of the time, if there are no registered extensions the default is an empty string "".

Reply:
Where do you get 80% of the time? With these changes, if there's any config to return, it'll be included in the payload. The code that formats these options is here: https://github.com/Azure/azure-functions-host/blob/d1067c54b27a8bb491867d692c9f7154f50552b8/src/WebJobs.Script/HostOptionsProvider.cs#L37. As you can see, hostConfig will at minimum be an empty JObject.

Reply:
Ah meant to say 98% of time (statistic from the query in description) -- thanks for the pointer. Makes sense double checked that we handle empty JObject in scale controller code